x/stakeibc/keeper/icqcallbacks_fee_balance.go

```diff
@@ -77,9 +78,10 @@ func FeeBalanceCallback(k Keeper, ctx sdk.Context, args []byte, query icqtypes.Q
 		rewardsCollectorAddress.String(),
 		clienttypes.Height{},
 		timeout,
+		"",
 	)
 
-	msgs := []sdk.Msg{transferMsg}
+	msgs := []proto.Message{transferMsg}
```

Comment:
Why the migration from `sdk.Msg` to `proto.Message`?

Reply:
icatypes.SerializeCosmosTx [https://github.com/notional-labs/stride/blob/sdk_47/x/stakeibc/keeper/msg_server_submit_tx.go#L288](ref) now take proto.message as parameter instead of sdk.Msg, so i migrated it.

Reply:
( https://github.com/notional-labs/stride/blob/sdk_47/x/stakeibc/keeper/msg_server_submit_tx.go#L288 )